kubernetes-extension-test/src/test/java/org/ballerinax/kubernetes/test/samples/Sample17Test.java

```diff
@@ -117,7 +134,7 @@ public void validateRoute() {
         Assert.assertEquals(route.getMetadata().getNamespace(), "bal-oc", "Namespace is missing.");
     
         Assert.assertNotNull(route.getSpec(), "Spec is missing.");
-        Assert.assertEquals(route.getSpec().getHost(), "helloep-openshift-route-bal-oc.<MINISHIFT_IP>.nip.io",
+        Assert.assertEquals(route.getSpec().getHost(), "helloep-openshift-route-bal-oc.192.168.99.131.nip.io",
```

Comment:
ip is hard coded here?

Reply:
This is for a test. Original content is replace back on "cleanUp" method.